CHANGELOG.md

```diff
@@ -35,7 +37,7 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
     relative to the last result received.
   * config: add new controller max_page_size field for controlling the default and max size
     of pages when paginating through results.
-
+    
```

Comment:
😬 mind removing the extra tab?

Reply:
haha got ya

Reply:
I think you accidentally removed the wrong tab?

Reply:
sorry about that @johanbrandhorst my editor insist on making that URL markdown information stay on the same line.  Let me know if that is what you were referring to.

Reply:
Ahh see it now